### DISCUSSION TITLE
Naming scheme at PDB2PQR online webserver

### DISCUSSION THREAD

**Discussion by srdjan125** [Electrostatics/pdb2pqr#310] [Q&A]
Hello,

I don't know if this is the right place to ask this question - if not please redirect me to another suitable forum.
While protonating protein at desired pH with Propka and online webserver, I can choose between Amber/CHARMM force field etc. After that, my goal is to use PQR files to calculate Lennard-Jones interaction between proteins.

For that, I have found atom types for CHARMM force field, and also created GROMACS topology files. 
The thing is that for example CB atoms from PQR files can correspond either to CT2, CT2A or CT3 atom type - all those 3 atom types have different LJ parameters, so I cannot be sure if  CB atom from PQR file is CT2, CT2A or CT3 atom type.
  Things are further complicated that GROMACS creates topology files where order of atoms of each aminoacid is different than in PQR files.
  Differentiating atoms by masses and partial charges also does not work, as some nitrogen or carbon atoms might have the same partial charge but different LJ parameters. 
  (for now the only way it seems to write python script to tell to GROMACS to protonate aminocid residues according to PDB2PQR and Propka assingment - and get topology files having same number of atoms like PQR files). 

 Could someone explain me how are atoms assigned names at PDB2PQR webserver? Maybe there should be some rules to determine which PQR atom has which LJ type? 

Thanx a lot.

**Comment by sobolevnrm** [comment]
Hello -- I'm going to move this question to the [PDB2PQR GitHub repo](https://github.com/Electrostatics/pdb2pqr) and answer it there.  Thank you.

**Comment by sobolevnrm** [comment]
This section of the user manual describes how residues/atoms are named and how force fields are assigned:  https://pdb2pqr.readthedocs.io/en/latest/extending.html

The relevant files can be found in https://github.com/Electrostatics/pdb2pqr/tree/master/pdb2pqr/dat

**Comment by srdjan125** [comment]
Hello,

Thank you for your help.
  I have already wrote python program to tell to GROMACS to protonate proteins according to Propka (by typing 0 or 1 each time when GROMACS asks what kind of protonation state I want for some residue).
   But now I see that using files CHARMM.dat or Amber.dat enables to see which PQR atom corresponds to which CHARMM/Amber atom type.

   I also see that CHARMM.dat is for Charmm27 version, which had only HA or HB atom types, but CHARMM36 has HA1, HA2,  HA3, HB1 or HB2 atom types.

   Greetings.
________________________________
